src/MongoDB/Command.c

```diff
@@ -74,9 +74,9 @@ static bool php_phongo_command_init(php_phongo_command_t *intern, zval *filter,
 
 	if (bson_iter_init(&iter, intern->bson) && bson_iter_find_descendant(&iter, "cursor.batchSize", &sub_iter) && BSON_ITER_HOLDS_INT(&sub_iter)) {
 		int64_t batch_size = bson_iter_as_int64(&sub_iter);
-		
+
 		if (batch_size >= 0 && batch_size <= UINT32_MAX) {
-			intern->batch_size = (uint32_t) batch_size;
+			intern->batch_size = (uint32_t)batch_size;
```

Comment:
We can chose to have the space after the cast to be inserted here as well, if we want. I would suggest to do this in a separate PR then, to see whether we want to.

Reply:
Ah, I commented on that earlier. Space after the cast is preferred for me.

Would it not make sense to do it in the same PR, since it's our existing convention and would limit the diff here?